tests/test_unittest/test_gtc/test_oir.py

```diff
@@ -30,3 +40,36 @@ def test_no_horizontal_offset_allowed():
 def test_mask_must_be_bool():
     with pytest.raises(ValidationError, match=r".*must be.* bool.*"):
         MaskStmtFactory(mask=FieldAccessFactory(dtype=DataType.INT32))
+
+
+def test_temporary_dimensions():
+    temp1d = Temporary(name="b", dtype=DataType.INT64, dimensions=(True, False, False))
+    assert temp1d.dimensions == (True, False, False)
+
+
+def test_assign_to_ik_fwd():
+    out_name = "ik_field"
+    in_name = "other_ik_field"
+    with pytest.raises(ValidationError, match=r"Not allowed to assign to ik-field"):
```

Comment:
I'd like to understand how this complements `test_lvalue_dims_validation` in `test_common.py`.

Reply:
I actually haven't contributed to none of those tests so I would also like to know... I think this comes from your initial work in this part. Should we delete this test?

Reply:
That's fine with me. @DropD?